addon/helpers/inline-styles.ts

```diff
@@ -0,0 +1,30 @@
+import { helper } from '@ember/component/helper';
+import { assert } from '@ember/debug';
+import cssMappings from '../utils/css-mappings';
+
+export function inlineStyles(_:unknown, params:Record<string, unknown>) {
```

Comment:
To avoid having to cast, create the interface and use it to type the `params` and remove the `as string` castings.
```suggestion
interface InlineStylesParams {
  componentName?: string;
  [property: string]: unknown;
}

export function inlineStyles(_: unknown, params: InlineStylesParams) {
```